src/pixee-platform.ts

```diff
@@ -5,14 +5,20 @@ import FormData from "form-data";
 import { TOOL_PATH } from "./inputs";
 import {getGitHubContext, getRepositoryInfo} from "./github";
 
-export async function uploadInputFile(tool: TOOL_PATH, file: string) {
-  const fileContent = fs.readFileSync(file, "utf-8");
+export async function uploadInputFile(tool: TOOL_PATH, files: Array<string>) {
   const form = new FormData();
-  form.append("file", fileContent);
-  const pixeeUrl = core.getInput("pixee-api-url");
 
+  // Append each file to the form data
+  files.forEach(file => {
+    const fileContent = fs.readFileSync(file);
+    form.append("files", fileContent, file);
```

Comment:
I think this is using the entire path and we might want to use just the base filename?

Reply:
i have addressed this